edge-apps/grafana/src/main.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  setupTheme,
+  getSetting,
+  getSettingWithDefault,
+  signalReady,
+  getAccessToken,
+} from '@screenly/edge-apps'
+import { getRenderUrl, fetchAndRenderDashboard } from './render'
+
+function displayError(message: string): void {
```

Comment:
Why don't we use panic overlay library? See: https://github.com/Screenly/Playground/blob/master/edge-apps/powerbi/static/js/main.js#L86

It shouldn't be a part of the Edge App. Later, we can use our own implementation and put it into '@screenly/edge-apps'